components/multiselect/observable-data.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Observable Data
+page_title: MultiSelect with Observable Data
+description: MultiSelect with Observable Data
+slug: multiselect-observable-data
+tags: telerik,blazor,multiselect,observable,data,collection
+published: True
+position: 25
+---
+
+# MultiSelect with Observable Data
+
+@[template](/_contentTemplates/common/observable-data-intro.md#observable-data-intro)
+
+
+>caption Bind the MultiSelect component to an [ObservableCollection]({%slug common-features-observable-data%}), so it can react to collection changes.
```

Comment:
Maybe the link here is not necessary, the article would not provide more details, but can confuse people and send them to read other articles too early.